backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -70,4 +94,19 @@ public void testGetWaitingTime() throws Exception {
         
         assertEquals("Waiting Time per customer is", WAITING_TIME_MINS, waitingTime);
     }
+
+    @Test
+    public void testGetTokens_forShop_success() throws Exception {
+        // Arrange
+        // Add Shop
```

Comment:
Not adding any value. Can be removed? Add comment only if you feel it'll add some additional value.

Reply:
yes removed!